Add rs-pug to Music and Media (#328)
ratatui/awesome-ratatui: README.md

- [O₂](https://github.com/coignard/o2) - Rust port of the ORCΛ esoteric programming language and terminal livecoding environment.
- [oosc-rs](https://github.com/karasikq/oosc-rs) - An additive wavetable synthesizer for terminal.
- [roon-tui](https://github.com/TheAppgineer/roon-tui) - Roon Remote for the terminal.

> - [rs-pug](https://github.com/JustRoccat/rs-pug) - A Neovim-inspired, asynchronous music player using mpv and yt-dlp, featuring a Lua plugin system and a reactive audio visualizer.

Reviewer: let's simplify the description a bit

```suggestion
- [rs-pug](https://github.com/JustRoccat/rs-pug) - A Neovim-inspired asynchronous music player with a reactive audio visualizer.
```

Author: sure
